### DISCUSSION TITLE
essay/2024/07/17/not-easy-but-not-hard-either

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#221] [Comments]
# essay/2024/07/17/not-easy-but-not-hard-either

컴퓨터 공부를 처음 시작한 날이 2004년 8월 25일. 이제 20년이 지났습니다.

https://jeho.page/essay/2024/07/17/not-easy-but-not-hard-either.html

**Comment by asyncFlow-dad** [comment]
멋지십니다

**Reply by BenjaminKim** [reply]
하하 그런가요? 고맙습니다. 😁

**Comment by osangu** [comment]
노력은 그 자체로도 가치있지만,  불확실한 미래를 향해 나아가는 결의가 있기에 더욱 빛 나는 것 같습니다.

항상 좋은 글 감사합니다!
점심 시간이 끝나고 워밍업 느낌으로 읽지만, 많은 생각의 빌미를 얻어가네요  : o

**Reply by BenjaminKim** [reply]
워밍업 잘 되셨으면 좋겠네요. 오늘도 힘내서 열심히 해보자구요!

**Reply by BenjaminKim** [reply]
저도 열심히 코딩 중입니다. ㅎ

**Comment by lee-dongyeop** [comment]
> 도대체 어느 정도 힘들어야 죽어버리는 게 낫겠다는 생각을 할 수 있을까?
>
> 저에게는 그런 날은 하루도 없었던 것입니다.
> 이 정도면 뭐 할만한 것 아닌가?

위 부분.. 정말 멋있습니다. 잘 봤어요!

**Reply by BenjaminKim** [reply]
고맙습니다! 😄
